pkgs/cli/src/main.zig

```diff
@@ -122,7 +128,7 @@ pub fn main() !void {
         },
         .prove => |provecmd| {
             std.debug.print("distribution dir={s}\n", .{provecmd.dist_dir});
-            const logger = utilsLib.getLogger(null);
+            const logger = utilsLib.getLogger(null, null, null);
```

Comment:
combine these two into one params struct that is passed or null

Reply:
also add another param into the last struct which is activeLogLevel for the file, call this struct FileParams 

also add a corresponding fileActiveLogLevel cli arg default set to debug, and change the logger's console active default level to info (also take it as a cli param with default info if thats not already there)